hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/app.tsx

```diff
@@ -27,6 +27,8 @@ import { routesV2 } from '@/v2/routes-v2';
 import { MakeRouteWithSubRoutes } from '@/makeRouteWithSubRoutes';
 import classNames from 'classnames';
 
+import Loader from '@/v2/components/loader/loader';
+
 import './app.less';
 
```

Comment:
In the UI, both the Columns dropdown and the Limit dropdown have downward-pointing arrows that allow us to select values. However, I've noticed an inconsistency: when I click anywhere on the Columns box, the dropdown appears correctly, but when I try clicking anywhere on the Limit box, the dropdown doesn't show up. It only appears when I click directly on the arrow. Could you please look into this and enable the dropdown to be triggered by clicking anywhere on the Limit box? I understand this isn’t a critical change, but it would contribute to a smoother user experience.

<img width="1650" alt="image" src="https://github.com/user-attachments/assets/424e94b9-f79c-40cc-8555-87d0f61fe6f3">

Reply:
Hi @ArafatKhan2198, thanks for pointing this out.
Fixed in [4947f05](https://github.com/apache/ozone/pull/7048/commits/4947f05588005938682fad69859a875a2141e2bf)